tap_twitter_ads/schemas/line_items.json

```diff
@@ -8,9 +8,6 @@
     "bid_type": {
       "type": ["null", "string"]
     },
-    "bid_strategy": {
-      "type": ["null", "string"]
-    },
```

Comment:
Why did we remove this field from the line_items? The line_items is not used in cards.

Reply:
Added back the `bid_strategy` field.